arch/posix/core/posix_core.c

```diff
@@ -283,10 +283,12 @@ static void *posix_thread_starter(void *arg)
 	/*
 	 * The program may have been finished before this thread ever got to run
 	 */
+	/* LCOV_EXCL_START */ /* See Note1 */
```

Comment:
That's interesting pragma. Will it mark the code inside as not covered, or as "not subject to coverage" ? I.e. would such blocks preclude achieving 100% of coverage?

Reply:
I can only refer you to the lcov documentation: http://ltp.sourceforge.net/coverage/lcov/geninfo.1.php

gcovr was supposed to support this pseudo pragmas too.

Reply:
the coverage reports we have and shown above do not use gcovr. gcovr is only used with shippable.